lib/web/integrations_awsra.go

```diff
@@ -140,3 +144,56 @@ func (h *Handler) awsRolesAnywhereConfigureTrustAnchor(w http.ResponseWriter, r
 
 	return nil, trace.Wrap(err)
 }
+
+// awsRolesAnywhereListProfiles lists profiles Roles Anywhere Profiles accessible by the integration.
```

Comment:
Can we clean up the wording here? _list profiles roles anywhere profiles_ doesn't sound right.